frontend/app/src/scenes/Home/scenes/AddCategory/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Button,
+  Col,
+  Form,
+  Input,
+  notification,
+  Row,
+  Spin,
+  Typography,
+} from 'antd';
+import mainStyles from '../../styles.css';
+import { AddCategoryStateProps } from './interfaces';
+import { RouteComponentProps, withRouter } from 'react-router';
+import axios, { AxiosResponse } from 'axios';
+import { Category } from '../../interfaces';
+import { handleApiError } from '../../../../services/util/errorHandler';
+
+const { Title, Text } = Typography;
+
+class AddCategory extends React.Component<
+  RouteComponentProps,
+  AddCategoryStateProps
+> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      isLoading: false,
+    };
+  }
+
+  onFinish = (values: any) => {
+    this.setState({ isLoading: true });
+    axios
+      .post(`${window.location.origin}/core/academix/admin/categories`, values)
```

Comment:
Let's use + to concatenate these types of strings

Reply:
Sure! Is there a reason to use + concatenation here?

Reply:
Resolved with commit [1398e95](https://github.com/sef-global/sef-core/pull/75/commits/1398e959bfe548ee997ef26decd68c7c751f96f7)